boards/xtensa/intel_s1000_crb/Kconfig.defconfig

```diff
@@ -167,6 +145,38 @@ config GPIO_DW_0
 
 endif
 
+if FLASH
+
+config SPI
+	def_bool y
+
+config SPI_NOR
+	def_bool y
+
+config SPI_NOR_JEDEC_ID
+	default 0xC2803A
+
+config SPI_NOR_PAGE_SIZE
+	default 256
+
+config SPI_NOR_SECTOR_SIZE
+	default 4096
+
+config SPI_NOR_SECTORS
+	default 1024
+
+choice SPI_NOR_BLOCK_ERASE_SIZE
```

Comment:
According to the datasheet, the block erase size is 64K. In any case, this parameter should not be a Kconfig variable.  Instead, this parameter should be determined based on SFDP.

Reply:
@sathishkuttan Note that the SPI NOR driver is used here which is intentionally using dts defines for flash dimensions over SFDP (code size & complexity). There is also a SFDP driver PR (#7561) but this one is not yet merged.

Reply:
Yes. There's a different PR for that as pointed out by @vanwinkeljan. Having said that, we can use the one specified by DTS for erase size and remove the KConfig one. I wanted to do it as a different PR but now that you have pointed it out, i'll try including that change in this PR itself.